src/Main.java

```diff
@@ -0,0 +1,21 @@
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) throws IOException {
```

Comment:
have you considered any error checking? e.g. user inputs fewer than 3 arguments, command name is not "search" etc.